src/MainnetController.sol

```diff
@@ -160,7 +163,7 @@ contract MainnetController is AccessControl {
         );
     }
 
-    function burnUSDS(uint256 usdsAmount) external onlyRole(RELAYER) isActive rateLimited(LIMIT_USDS_BURN, usdsAmount) {
+    function burnUSDS(uint256 usdsAmount) external onlyRole(RELAYER) isActive cancelRateLimit(LIMIT_USDS_MINT, usdsAmount) {
```

Comment:
I don't think we should allow this to not have any limits (`swapUSDCToUSDS` as well).
I think it should also have it's own rateLimit, that would be increased by minting.
Better safe than sorry and also we can always set to unlimited anyway.

Reply:
Yeah I think this is a good call

Reply:
I don't think this is a matter of better safe than sorry, and instead this will make us more susceptible to DOS-type attacks.

There are cases where you want rate limits in both directions such as the PSM3 you want limits on deposit (safety in case of extreme market conditions) and withdraw (depegs are not easy to do by relayer).

In the case of mint/burn though this is a clear one cancels out the other. PSM may require a bit more thought, but I don't think there is any issue with swapping as much USDC as possible to USDS. The other direction is of course dangerous because you can depeg the PSM.